### ISSUE TITLE
Show Elixir attributes to handler

### ISSUE THREAD

**Issue by opqdonut** [CSCfi/rems#2157]
After #2130 and #2142, we could show some Elixir attributes to the handlers, e.g.
- bona fide status
- organization id

**Comment by opqdonut** [comment]
Rendering the bona fide status was implemented as part of #2142 in PR #2403.

User organizations are already rendered, although a separate ticket might be needed for reading organization membership from GA4GH AffiliationAndRole Visas.
